examples/step-87/doc/intro.dox

```diff
@@ -0,0 +1,273 @@
+<br>
+
+<i>
+This program was contributed by Magdalena Schreter-Fleischhacker
+and Peter Munch. Many ideas presented here are the result of common code
+development with  Maximilian Bergbauer, Marco Feder,
+Niklas Fehn, Johannes Heinz, Luca Heltai, Martin Kronbichler, Nils Much,
+and Judith Pauen.
+
+This tutorial is loosely based on chapter 3.4 of the submitted
+PhD thesis "Matrix-free finite element computations at extreme scale and for
+challenging applications" by Peter Munch. Magdalena Schreter-Fleischhacker
+is funded by the Austrian Science Fund (FWF) Schrödinger Fellowship  (J4577).
+</i>
+
+<h1>Introduction</h1>
+
+This tutorial presents the advanced point-evaluation functionalities of
+deal.II, specifically useful for evaluating finite element solutions at
+arbitrary points. The underlying finite element mesh can be distributed
+among processes, which makes the operations more involved due to communication.
+In the examples discussed in this tutorial, we focus on point evaluation for
+MPI-parallel computations, like parallel::distributed::Triangulation.
+Nevertheless, the application to non-distributed meshes is also possible.
+
+<h3>%Point evaluation</h3>
+
+In the context of the finite element method (FEM), it is a common task to
+query the solution $u$ at an arbitrary point $\boldsymbol{x}_q$ in the
+domain of interest $\Omega$
+@f[
+u(\boldsymbol{x}_q) = \sum_{i} N_i(\boldsymbol{x}_q) u_i  \quad\text{with}\quad i\in[0,n_{\text{dofs}}),
+@f]
+by evaluating the shape functions $N_i$ at this point  together with the corresponding  solution coefficients $u_i$.
+After identifying the cell $K$ where the point $\boldsymbol{x}_q$ resides, the
+transformation between $\boldsymbol{x}_q$ and the corresponding coordinates in the reference cell $\hat{\boldsymbol{x}}_q$ is obtained by the mapping $\boldsymbol{x}_q=\boldsymbol{F}_K(\hat{\boldsymbol{x}}_q)$.  In this setting, the evaluation of the solution at an arbitrary point boils down to a cell-local evaluation
```

Comment:
Here and elsewhere, it would be good to break the lines around 80 characters. I think there is no strict rule outside `.cc` files, but I find it much easier to work with code, e.g. read diffs, with hard line breaks.